ext/sodium/libsodium.stub.php

```diff
@@ -165,75 +230,116 @@ function sodium_crypto_secretstream_xchacha20poly1305_pull(string &$state, strin
 function sodium_crypto_secretstream_xchacha20poly1305_rekey(string &$state): void {}
 #endif
 
+/** @sensitive-param $key */
 function sodium_crypto_shorthash(string $message, string $key): string {}
 
 function sodium_crypto_shorthash_keygen(): string {}
 
+/** @sensitive-param $secret_key */
 function sodium_crypto_sign(string $message, string $secret_key): string {}
 
+/** @sensitive-param $secret_key */
 function sodium_crypto_sign_detached(string $message, string $secret_key): string {}
 
 function sodium_crypto_sign_ed25519_pk_to_curve25519(string $public_key): string {}
 
+/** @sensitive-param $secret_key */
 function sodium_crypto_sign_ed25519_sk_to_curve25519(string $secret_key): string {}
 
 function sodium_crypto_sign_keypair(): string {}
 
+/** @sensitive-param $secret_key */
 function sodium_crypto_sign_keypair_from_secretkey_and_publickey(string $secret_key, string $public_key): string {}
 
 function sodium_crypto_sign_open(string $signed_message, string $public_key): string|false {}
 
+/** @sensitive-param $key_pair */
 function sodium_crypto_sign_publickey(string $key_pair): string {}
 
+/** @sensitive-param $key_pair */
 function sodium_crypto_sign_secretkey(string $key_pair): string {}
 
+/** @sensitive-param $secret_key */
 function sodium_crypto_sign_publickey_from_secretkey(string $secret_key): string {}
 
+/** @sensitive-param $seed */
 function sodium_crypto_sign_seed_keypair(string $seed): string {}
 
 function sodium_crypto_sign_verify_detached(string $signature, string $message, string $public_key): bool {}
```

Comment:
here as well

Reply:
This is effectively the inverse of sign: You received a public message and want to verify its authenticity. For messages that are signed and encrypted the "first encrypt and then sign the ciphertext (Encrypt Then MAC)" construction is the most secure and for that one the given `$message` would be the ciphertext.

Reply:
Thanks!